package.json

```diff
@@ -18,6 +18,7 @@
   "homepage": "https://github.com/Meghajit/Papyrus#readme",
   "dependencies": {
     "express": "4.17.1",
+    "multer": "^1.4.2",
```

Comment:
Can we remove the `^` ? Else it will trigger automatic version upgrades for minor and patch versions.
I am not sure if we can guarantee backward compatibility atm, since tests are missing.

Reply:
Adding a `^` sign makes sure that we keep all the `minor` and `patch` versions updated and not the major ones.
According to semantic versioning guidelines if its a minor version upgrade it should always be backwards compatible and not be a breaking change. So, I think it's safe (and actually better) to keep the dependencies updated for any of the users of this repo.